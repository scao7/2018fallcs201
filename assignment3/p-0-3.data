0 5 476 ; 4 0 345 ; 2 5 46 ; 0 7 287 ; 1 2 312 ; 
6 5 231 ; 4 4 311 ; 4 7 265 ; 5 6 90 ; 7 0 369 ; 
4 5 317 ; 4 2 238 ; 1 0 74 ; 0 3 189 ; 2 3 221 ; 
0 2 498 ; 4 5 0 ; 3 6 340 ; 1 1 345 ; 2 6 198 ; 
7 3 385 ; 1 1 305 ; 2 4 394 ; 5 1 319 ; 1 6 244 ; 
3 4 426 ; 1 0 26 ; 7 0 70 ; 0 5 469 ; 6 7 195 ; 
2 0 497 ; 3 6 352 ; 
