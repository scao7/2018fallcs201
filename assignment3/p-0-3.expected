0: 0
1: 1(0)74 3(0)189
2: 2(3)221
3: 4(2)238 5(2)46 6(2)198
4: 7(6)195
weight: 1161
